### PR TITLE
[MIG] sale_order_import_ubl: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
- Standard migration.
- Requires modules:
  + `sale_order_import` migrated in https://github.com/OCA/edi/pull/302
  + `base_ubl` migrated in https://github.com/OCA/edi/pull/303

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
Code review and Functional tests. LGTM

**Comment by dsolanki-initos** [comment]
Hi @phuctranerp can you please resolve the conflicts and ask someone for review to get the PR merge soon

**Comment by phuctranfxvn** [comment]
> Hi @phuctranerp can you please resolve the conflicts and ask someone for review to get the PR merge soon

Conflict resolved, thanks

**Comment by bosd** [comment]
Travis is stil red

**Comment by dsolanki-initos** [comment]
Hi @phuctranerp cau you please fix the travis?

**Comment by phuctranfxvn** [comment]
> Hi @phuctranerp cau you please fix the travis?

Test runs fine for this module. However, test case of another module (`base_business_document_import`), which has been merged into the 14.0 version, failed.

Related PRs: 
- https://github.com/OCA/edi/pull/301
- https://github.com/OCA/edi/pull/366

Updated: Travis fixed

**Comment by dsolanki-initos** [comment]
Travis is now fixed and now we can merge this PR. Can someone please do it?

**Comment by dsolanki-initos** [comment]
It would be great if we could merge the PR.

**Review by simahawk — CHANGES_REQUESTED** [review]
you cannot, it depends on https://github.com/OCA/edi/pull/302
and test requirements should be cleaned before merging.

**Comment by dsolanki-initos** [comment]
@phuctranerp can you please resolve conflicts so that we can merge the PR ASAP.

**Comment by phuctranfxvn** [comment]
Hi @dsolanki-initos

Conflicts fixed, we need to wait for #302 to be merged and re-run the travis to have it green

**Comment by dsolanki-initos** [comment]
@phuctranerp  the depended modules PR is merged can you please update this branch so that we can merge this PR?

**Comment by phuctranfxvn** [comment]
> @phuctranerp can you please resolve conflicts so that we can merge the PR ASAP.

I re-run travis check and ci/runbot. Should be ok now.

**Comment by dsolanki-initos** [comment]
It would be great if we could get the reviews so that we can merge the PR asap.

**Comment by dsolanki-initos** [comment]
It would be great if we could get the reviews so that we can merge the PR asap.

**Comment by dsolanki-initos** [comment]
Hi @jgrandguillaume 
It would be nice if you could add the reviews here.
Thank You!

**Comment by simahawk** [comment]
@TDu can you have a look?

**Comment by simahawk** [comment]
> It would be great if we could get the reviews so that we can merge the PR asap.

FTR there's no ASAP here. There's just "who has time and knowledge to review" :wink: 

Meanwhile, you can survive w/ pending merges using git-aggregator or installing via pip straight from the PRs.

File: sale_order_import_ubl/wizard/sale_order_import.py
Comment (TDu) [inline sale_order_import_ubl/wizard/sale_order_import.py]: I don't understand this change, `ubl_parse_delivery_details` has been merged into 13.0 https://github.com/OCA/edi/pull/349/files
The commit is present in the 14 branch but it does not have all its content. Indeed the part that was meant for `base_ubl` has been left aside.
This is why we have this change now.

In my opinion this commit  should be properly recovered https://github.com/OCA/edi/pull/349/commits

Comment (dsolanki-initos) [reply sale_order_import_ubl/wizard/sale_order_import.py]: @phuctranerp Will you please check  above comments ?

Comment (hailangvn) [reply sale_order_import_ubl/wizard/sale_order_import.py]: Good day @TDu, Thanks for your review. The 13.0 PR has been brought forward to 14.0 by https://github.com/OCA/edi/pull/406. So this PR is currently depending on it. Please help to review both PR. Thank you in advance.

**Comment by dsolanki-initos** [comment]
Hi @phuctranerp Can you please check Travis?

**Comment by hailangvn** [comment]
Thanks @dsolanki-initos, @TDu, the travis is green now.

**Comment by dsolanki-initos** [comment]
can we merge this PR?

**Review by simahawk — APPROVED** [review]

**Pull request merged**
